libs/langchain/langchain/vectorstores/vectara.py

```diff
@@ -451,12 +450,13 @@ class VectaraRetriever(VectorStoreRetriever):
     """Vectara vectorstore."""
     search_kwargs: dict = Field(
         default_factory=lambda: {
-            "lambda_val": 0.025,
```

Comment:
out of curiosity what's the motivation for this?

Reply:
This controls hybrid search. It's not a massive difference, just thinking that users may not be aware that the default is some form of hybrid search.